tests/src/JIT/HardwareIntrinsics/X86/Sse2/MoveScalar.Int64.cs

```diff
@@ -0,0 +1,56 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+
+using System;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.X86;
+
+namespace IntelHardwareIntrinsicTest
```

Comment:
nit: It would probably be better if these were part of the template-generated tests, like the SSE (and most other ISAs) tests are.

Reply:
Agree with @tannergooding .

Additionally, for hand-written tests, we usually put all the base-type variances of an intrinsic in a single file.

Reply:
I agree template-generated tests are better for most cases. But in this case, a lot of SSE2 tests (e.g. MoveScalar(double)) are based on TestTableSse2.cs. If we use template-generated tests for MoveScalar(long/ulong), they will look different with MoveScalar(double). In addition, MoveScalar(long/ulong) probably needs a new template because it sets the lower 64 bits of dest based on src operand and leaves the higher 64 bits of dest to 0.

Reply:
> But in this case, a lot of SSE2 tests (e.g. MoveScalar(double)) are based on TestTableSse2.cs

Yes, SSE2 is currently the outlier where almost none of the tests are using the templates (this is in contrast to the other SSE/AVX ISAs where almost all of the tests are using templates).

> If we use template-generated tests for MoveScalar(long/ulong), they will look different with MoveScalar(double

This should be fine, there is a pending work item to move all the SSE2 tests over to the templates anyways.

>  In addition, MoveScalar(long/ulong) probably needs a new template because it sets the lower 64 bits of dest based on src operand and leaves the higher 64 bits of dest to 0.

This shouldn't be an issue. The templates are already set-up to handle the differences between scalar and vector operations. The entry would look similar to:
```
("SimpleUnOpTest.template",         new Dictionary<string, string> { ["Isa"] = "Sse2", ["LoadIsa"] = "Sse2", ["Method"] = "MoveScalar",                   ["RetVectorType"] = "Vector128", ["RetBaseType"] = "Int64",  ["Op1VectorType"] = "Vector128", ["Op1BaseType"] = "Int64",                                                                                                        ["LargestVectorSize"] = "16", ["NextValueOp1"] = "(long)(random.Next(int.MinValue, int.MaxValue))",                                                                                               ["ValidateFirstResult"] = "result[0] != firstOp[0]",                                                                                                            ["ValidateRemainingResults"] = "result[i] != 0"}),
```

Reply:
I am moving all non-template SSE2 test to templated ones. Therefore, it should not have any negative impact to have implemented either way as they will get unified anyway.

Reply:
Sounds good.

It seems we have a couple of options: (1) merge this PR and @4creators will refactor all the tests including this one (2) delete tests from this PR and use @4creators 's tests. Either way should be fine.